.travis.yml

```diff
@@ -2,6 +2,7 @@ language: python
 python:
   - 3.7
   - 3.8
+  - 3.9-dev
```

Comment:
Consider replacing this with:
```yaml
matrix:
  include:
  - python: 3.9-dev
    env: TEST_RUNNER=green
    script: green
    after_success: echo
```
instead so we skip the linting until mypy gets fixed?

Reply:
I edited the yaml above, use `make testall` instead, and drop the ENV. Copy Pasta mistake.

Reply:
That's good. I will try it !